packages/account/src/Styles/account.scss

```diff
@@ -1792,6 +1791,22 @@ $MIN_HEIGHT_FLOATING: calc(
         }
     }
 
+    &__container {
+        display: flex;
+
+        @include mobile {
+            flex-direction: column-reverse;
+        }
+    }
+
+    &__article {
+        margin-top: -4rem;
+
+        @include mobile {
+            margin-top: -0.8rem;
```

Comment:
why do we need this?

Reply:
It is to move the infobox's position to the same as what is given in the design. I've tried to use `margin-bottom: 4rem` but it doesn't work so have to use -4rem instead